Libraries/LogBox/Data/LogBoxData.js

```diff
@@ -23,6 +23,9 @@ import type {
 import parseErrorStack from '../../Core/Devtools/parseErrorStack';
 import type {ExtendedError} from '../../Core/Devtools/parseErrorStack';
 import NativeLogBox from '../../NativeModules/specs/NativeLogBox';
+
+const ExceptionsManager = require('../../Core/ExceptionsManager');
```

Comment:
This was originally in reportLogBoxError to avoid a circular dependency. If you move it back into reportLogBoxError, but simply add the mock, does it not work as expected?

Reply:
Thanks for taking a look.  In my experience this change is necessary to avoid Jest throwing:

> ReferenceError: You are trying to `import` a file after the Jest environment has been torn down.

I'm not sure how to resolve the circular dependency issue :\